api_docs/tf_quant_finance/rates.md

```diff
@@ -37,7 +37,7 @@ Functions to handle rates.
 
 ## Classes
 
-[`class SwapCurveBuilderResult`](../tf_quant_finance/rates/SwapCurveBuilderResult.md): SwapCurveBuilderResult(times, rates, discount_factors, initial_rates, converged, failed, iterations, objective_value)
+[`class SwapCurveBuilderResult`](../tf_quant_finance/rates/SwapCurveBuilderResult.md): Build curves for swapping future cash flows.
```

Comment:
How about: 

'Results from a swap curve construction.'

This is because actual curve fitting/building is done by swap_curve_fit method.

Instead of updating rates.md file, could you please navigate to [SwapCurveBuilderResult](https://github.com/google/tf-quant-finance/blob/210e9521584596143cfba80ad212cefdf2b8fab6/tf_quant_finance/rates/swap_curve_common.py#L19) and add

SwapCurveBuilderResult.__doc__ = """Results from a swap curve construction."""